test/Passes/DefaultPipeline/tpp-conversion.mlir

```diff
@@ -1,27 +1,5 @@
 // RUN: tpp-opt %s -tpp-conversion -split-input-file | FileCheck %s
 
-#map0 = affine_map<(d0, d1, d2, d3, d4, d5) -> (d0, d2, d3, d5)>
-#map1 = affine_map<(d0, d1, d2, d3, d4, d5) -> (d1, d2, d5, d4)>
-#map2 = affine_map<(d0, d1, d2, d3, d4, d5) -> (d0, d1, d3, d4)>
-
-// Rewriting to brgemm is done after bufferization which is why it is not part of 'tpp-mapping'.
-// See the default pipeline in 'DefaultTppPasses' for more details.
-func.func @generic_to_brgemm(%arg0: tensor<4x16x32x32xf32>, %arg1: tensor<8x16x32x32xf32>, %arg2: tensor<4x8x32x32xf32>) -> tensor<4x8x32x32xf32> {
```

Comment:
What's wrong with this test? It should still work, no?

Reply:
I removed from `tpp-conversion` two passes: `createConvertForAllToParallelOpPass` and `createRewriteToBatchReduceGemmPass`. Both of these require to run bufferization first. `createConvertForAllToParallelOpPass` will not work as scf.parallel works only at memref while `createRewriteToBatchReduceGemmPass` will emit `scf.for` and not `scf.parallel` at the tensor level. So this test does not work anymore as we don't rewrite to brgemm at tensor level (also the test comment what misleading - we are at tensor level and we map to brgemm).

Reply:
It's fine to remove it. Conversion of linalg.generic to tpp.brgemm is still covered by the next test case.

Reply:
done.